src/zenml/integrations/huggingface/steps/accelerate_runner.py

```diff
@@ -57,22 +57,30 @@ def training_pipeline(some_param: int, ...):
 
     Args:
         step_function: The step function to run.
-        num_processes: The number of processes to use.
-        use_cpu: Whether to use the CPU.
+        kwargs: Any additional arguments to pass to the accelerate arguments (num_processes, cpu, etc.).
 
     Returns:
         The accelerate-enabled version of the step.
+
+    Raises:
+        RuntimeError: If the decorator is misused.
+
     """
 
-    def _decorator(entrypoint: F) -> F:
+    def _decorator(entrypoint: F, accelerate_kwargs: Dict[str, Any]) -> F:
         @functools.wraps(entrypoint)
         def inner(*args: Any, **kwargs: Any) -> Any:
             if args:
                 raise ValueError(
                     "Accelerated steps do not support positional arguments."
                 )
 
-            if not use_cpu:
+            num_processes = None
+            if np := kwargs.get("num_processes", None):
```

Comment:
why is num_processes treated differently here than any other kwarg?

Reply:
It was just there, but you are right - I unified it to give back the control to the user.